### PR TITLE
macOS - tweak traffic light position handling

### PR DESCRIPTION
**Pull request by bpasero**
//cc @jrieken

This PR fixes #

### CODE REVIEW

**Review by ProKashif — APPROVED** [review]

**Review by chrmarti — APPROVED** [review]

File: src/vs/platform/windows/electron-main/window.ts [hunk 1]
```diff
@@ -1325,9 +1326,14 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 
 		const useCustomTrafficLightPosition = this.configurationService.getValue<boolean>(commandCenterSettingKey);
 		if (useCustomTrafficLightPosition) {
-			this._win.setTrafficLightPosition({ x: 7, y: 9 });
+			if (!this.defaultTrafficLightPosition) {
+				this.defaultTrafficLightPosition = this._win.getTrafficLightPosition(); // remember default to restore later
+			}
```
Comment (jrieken) [inline src/vs/platform/windows/electron-main/window.ts:1331]: @bpasero On first call this does return `{x:0, y:0}` which seems fishy and therefore I didn't persist the old value. Is the (0,0) a well-known value for default?

**Review by jrieken — COMMENTED** [review]
unsure how getTrafficLightPosition is spec'd

**Comment by deepak1556** [comment]
Yeah unless the `trafficLightPosition` option is set to a custom value when creating the `BrowserWindow` constructor, the default value of `getTrafficLightPosition` is `{0,0}`. The API implementation is a getter, it will return whatever value is set via `setTrafficLightPosition`.

**Comment by bpasero** [comment]
I filed https://github.com/electron/electron/issues/34969 and it was confirmed to be a bug.

**Pull request merged**